packages/react-radio/src/components/Radio/Radio.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { useRadio } from './useRadio';
+import { RadioProps } from './Radio.types';
+import { renderRadio } from './renderRadio';
+import { useRadioStyles } from './useRadioStyles';
+
+/**
+ * A Radio component presents a set of options where only one option can be selected.
+ */
+export const Radio = React.forwardRef<HTMLElement, RadioProps>((props, ref) => {
```

Comment:
Should we change this to `ForwardRefComponent<RadioProps>` to follow #19923 or are you planning on doing that in a separate PR?

Reply:
Ah I should have updated the template. But it would also be good if you could update the component in this PR.

Reply:
Updated the component in this PR. Thanks for pointing it out :)